examples/step-50/doc/intro.dox

```diff
@@ -67,27 +112,34 @@ implements what we will refer to as the "first-child rule" where, for each cell
 in the hierarchy, we recursively assign the parent of a cell to the owner of the
 first child cell. The following figures give an example of such a distribution. Here
 the left image represents the active cells for a sample 2D mesh partitioned using a
-space-filling curve (similar to p4est), the center image gives the tree representation
-of the active mesh, and the right image gives the multilevel hierarchy of cells. The
+space-filling curve (which is what p4est uses to partition cells);
+the center image gives the tree representation
+of the active mesh; and the right image gives the multilevel hierarchy of cells. The
 colors and numbers represent the different processors. The circular nodes in the tree
 are the non-active cells which are distributed using the "first-child rule".
 
-<img width="400px" src="https://www.dealii.org/images/steps/developer/step-50-workload-example.png" alt="">
+<img width="800px" src="https://www.dealii.org/images/steps/developer/step-50-workload-example.png" alt="">
 
 Included among the output to screen in this example is a value "Partition efficiency"
 given by one over MGTools::workload_imbalance(). This value, which will be denoted
 by $\mathbb{E}$,  quantifies the overhead produced by not having a perfect work balance
-on each level of the multigrid hierarchy (as is evident from the example above).
+on each level of the multigrid hierarchy. This imbalance is evident from the
+example above: while level $\ell=2$ is about as well balanced as is possible
+with four cells among three processors, the coarse
```

Comment:
Should we maybe add that this balance is *by chance*?

Reply:
Well, actually no. The finest level is *always* about as well balanced as one can.

Reply:
The *active* level is as balanced as it can be. But that does not imply that the cells on the finest level are balanced. Imagine what would happen if the first of the four level-1 cells in the figure is refined rather than the third. In that case, we have 3 cells on proc 0, 1 cell on proc 1 and none on proc 2. Or did I misread what you wanted to say?

Reply:
Oh, I see what you mean. 

I think to explain it at this level of detail would require a paragraph. It may not be necessary in the current context to make the point we're trying to make here. But I will if you insist!

Reply:
No, I don't insist, let's keep it this way, it is very good and it gets clear from the context if one really wants to know it.